helmfile.yaml

```diff
@@ -0,0 +1,71 @@
+repositories:
+  - name: gatekeeper
+    url: https://open-policy-agent.github.io/gatekeeper/charts
+  - name: ratify
+    url: https://deislabs.github.io/ratify
+    
+releases:
+  - name: gatekeeper
+    namespace: gatekeeper-system
+    createNamespace: true
+    chart: gatekeeper/gatekeeper
+    version: 3.12.0
+    wait: true
+    set:
+      - name: enableExternalData
+        value: true
+      - name: validatingWebhookTimeoutSeconds
+        value: 5
+      - name: mutatingWebhookTimeoutSeconds
+        value: 2
+  - name: ratify
+    namespace: gatekeeper-system
+    chart: ratify/ratify
+    version: 1.8.0
+    wait: true
+    needs:
+      - gatekeeper
+    hooks:
+      - events: ["presync"]
+        showlogs: true
+        command: "bash"
+        args:
+          - "-c"
+          - "kubectl apply -f https://deislabs.github.io/ratify/library/default/template.yaml && kubectl apply -f https://deislabs.github.io/ratify/library/default/samples/constraint.yaml"
+      - events: ["postuninstall"]
+        showlogs: true
+        command: "kubectl"
+        args:
+          - "delete"
+          - "-f"
+          - "https://deislabs.github.io/ratify/library/default/template.yaml"
+          - "--ignore-not-found=true"
+      - events: ["postuninstall"]
+        showlogs: true
+        command: "kubectl"
+        args:
+          - "delete"
+          - "-f"
+          - "https://deislabs.github.io/ratify/library/default/samples/constraint.yaml"
+          - "--ignore-not-found=true"
+      - events: ["postuninstall"]
+        showlogs: true
+        command: "kubectl"
+        args:
+          - "delete"
+          - "crd"
+          - "stores.config.ratify.deislabs.io"
+          - "verifiers.config.ratify.deislabs.io"
+          - "certificatestores.config.ratify.deislabs.io"
```

Comment:
probably we also need to delete policy crd as well.

Reply:
yeah good point. versioning gets tricky though since the current release doesn't yet have the policy CRD